sources/osgShadow/ShadowReceiveAttribute.js

```diff
@@ -65,7 +69,7 @@ define( [
         attributeType: 'ShadowReceive',
 
         cloneType: function () {
-            return new ShadowReceiveAttribute( this._lightNumber, true );
+            return new ShadowReceiveAttribute( this._light, true );
```

Comment:
you sure it's debug code ?
I tought of that as real code, when user change lightnum of its light ?